src/views/juttle-view.js

```diff
@@ -77,19 +77,15 @@ var JuttleView = Base.extend({
         }
     },
 
-    _consume_mark: function(time) {}, //to be overridden as part of the consume_mark template pattern defined below
-    consume_mark: function(time) {
-        if (_.isString(time)) {
```

Comment:
I removed it in a separate commit. In Jut 1.0 it was necessary because dates came across the wire as strings, but with jsdp, they'll get converted into real dates before being sent here. In summary, I thought this check was unnecessarily lax.